pkg/occlient/occlient.go

```diff
@@ -1672,7 +1673,9 @@ func (c *Client) StartBuild(name string) (string, error) {
 }
 
 // WaitForBuildToFinish block and waits for build to finish. Returns error if build failed or was canceled.
-func (c *Client) WaitForBuildToFinish(buildName string) error {
+func (c *Client) WaitForBuildToFinish(buildName string, stdout io.Writer) error {
+	// following indicates if we have already setup the following logic
+	following := false
 	glog.V(4).Infof("Waiting for %s  build to finish", buildName)
 
 	w, err := c.buildClient.Builds(c.Namespace).Watch(metav1.ListOptions{
```

Comment:
I understand this is not touched in the proposed PR but can we add a comment that describes what this all about? At least I didn't understand a thing here. :disappointed: